clang/lib/CodeGen/CGVTables.cpp

```diff
@@ -124,7 +124,7 @@ static void resolveTopLevelMetadata(llvm::Function *Fn,
   auto *DIS = Fn->getSubprogram();
   if (!DIS)
     return;
-  auto *NewDIS = DIS->replaceWithDistinct(DIS->clone());
+  auto *NewDIS = llvm::MDNode::replaceWithDistinct(DIS->clone());
```

Comment:
drive-by?

Reply:
Yep, drive-by. I was looking up how others used the clone method, and was initially confused by this place here, because it called a static method via an object pointer.

Will update the commit message to mention that this is a drive by NFC